Site/.env.local.example

```
@@ -0,0 +1,33 @@
+BROWSERSLIST_ENV=modern
+BROWSERSLIST_CONFIG=../../install/.browserslistrc
+
+# A valid prerelease (QA) login
+VEUPATHDB_LOGIN_USER=
+VEUPATHDB_LOGIN_PASS=
+
+ROOT_ELEMENT=#wdk-container
+ROOT_URL=/a/app
+LEGACY_WEB_APP_ENDPOINT=/a
+LEGACY_WEB_APP_URL=https://qa.clinepidb.org/ce.b19
+
+PROJECT_ID=ClinEpiDB
+USE_EDA=true
```

It might be nice to prefix all eda-related variables.
```suggestion
EDA_ENABLED=true
```